### ISSUE TITLE
Bug: "Deploy FLIR" without function 

### ISSUE THREAD

**Issue by Roman-laufenderWahnsinn** [Project-Hatchet/public_h-60#403]
We have the problem, that random, after some time non of the pilots can unstow the FLIR.
We can open the FLIR Page, but clicking on "Deploy FLIR" does nothing.
This error is not there after a Server restart and does not happen, if only a few pilots test it out.
From the moment we have the deploy problem, non can boot up the FLIR, but every Pilot, who has the FLIR open and functional, will stay fully functional.
Here is an RTP File from a session after a Training. The error occurred directly after spawning the bird and therefore i assume the "File z\vtx\addons\uh60_jvmf\functions\fnc_ctabToJvmf.sqf..., line 62" may be part of the equation.
https://gist.github.com/Roman-laufenderWahnsinn/2b3989c25608ced3e4cef30db998a6d3

**Comment by Roman-laufenderWahnsinn** [comment]
I recorde the Problem for better untersanding:
https://streamable.com/he9ndq

The RTP File of the recording:
https://gist.github.com/Roman-laufenderWahnsinn/9eee905b1ef3424f5ef958e6c94b5b50

Hope this helps.

Edit: a clan member pointed somthing out to me, we suspected cTab, but for me it didnt sound logical. BUT we use SIT (https://steamcommunity.com/workshop/filedetails/?id=2262009445) and the error is since we included cTab and SIT. Mod incompatible?

**Comment by BroBeansCPG** [comment]
Unlikely it’s related to sit, 

But we’re tracking a similar issue with the FLIR having issues deploying

**Comment by Roman-laufenderWahnsinn** [comment]
I have read the other issue.
I opened this issue, because the other one can be reproduced and is different from this problem. We have had situations like the other issue, but the FLIR deployed, only took more time.

We will try not using cTab and/or sit and if we run better I will further investigate.

**Comment by Roman-laufenderWahnsinn** [comment]
Update: we have pined down the source of the error:
We added cusom icons to cTab and therfore the count is different and the switch case for "salute", "medical" and "other" does not work properly.
So this is a "us" bug.
I think this can be closed.

**Comment by BroBeansCPG** [comment]
> Update: we have pined down the source of the error: We added cusom icons to cTab and therfore the count is different and the switch case for "salute", "medical" and "other" does not work properly. So this is a "us" bug. I think this can be closed.

What version of cTab were you using?

**Comment by BroBeansCPG** [comment]
Duplicate of #420

**Comment by Roman-laufenderWahnsinn** [comment]
We used one, where we added custom icons with ids ranging in the 5xxx. As this is far out of the cTab to jfmv funktion, that created the bug. Am 27.07.23, 06:31 schrieb BroBeans ***@***.***>:

   
   
   Update: we have pined down the source of the error: We added cusom icons to cTab and therfore the count is different and the switch case for "salute", "medical" and "other" does not work properly. So this is a "us" bug. I think this can be closed. 
   
  What version of cTab were you using?
